payjoin/tests/integration.rs

```diff
@@ -205,8 +206,9 @@ mod integration {
                 let ohttp_relay = services.ohttp_relay_url();
                 let mock_address = Address::from_str("tb1q6d3a2w975yny0asuvd9a67ner4nks58ff0q8g4")?
                     .assume_checked();
-                let mut bad_initializer =
-                    Receiver::new(mock_address, directory, bad_ohttp_keys, None)?;
+                let new_receiver = NewReceiver::new(mock_address, directory, bad_ohttp_keys, None)?;
+                let storage_token = new_receiver.persist(&mut NoopPersister)?;
```

Comment:
nit: these can be chained since the "new_receiver" variable isn't really useful 

```suggestion
                let storage_token = NewReceiver::new(
                    address.clone(),
                    directory.clone(),
                    ohttp_keys.clone(),
                    Some(Duration::from_secs(0)),
                )?
                .persist(&mut NoopPersister)?;
```